src/ontology/uberon-edit.obo

```diff
@@ -118175,7 +118176,7 @@ xref: ANISEED:1235299
 xref: BSA:0000064
 xref: BTO:0003173
 xref: Wikipedia:Endostyle
-is_a: UBERON:0006846 ! surface groove
+is_a: UBERON:0003102 ! surface structure
```

Comment:
'surface groove', once immaterial, seems appropriate.

Reply:
I changed it as definition: An endostyle is a longitudinal ciliated groove on the ventral wall of the pharynx which produces mucus to gather food particles.

If something produces mucus, it should be material - that being said I haven't read into it, im going fully based on the definition

Reply:
> If something produces mucus, it should be material 

Seems reasonable. It seems the challenge is how one models the idea that an "organ" (as it is described in the wiki link), which has the shape of a groove, contains mucus-secreting glands.

Reply:
> which has the shape of a groove

Perhaps:
`'has characteristic' some grooved`?